alternator/executor.cc

```diff
@@ -3258,12 +3323,15 @@ future<executor::request_return_type> executor::get_item(client_state& client_st
     auto attrs_to_get = calculate_attrs_to_get(request, used_attribute_names);
     const rjson::value* expression_attribute_names = rjson::find(request, "ExpressionAttributeNames");
     verify_all_are_used(expression_attribute_names, used_attribute_names, "ExpressionAttributeNames", "GetItem");
+    bool ac = should_add_capacity(request);
 
     return _proxy.query(schema, std::move(command), std::move(partition_ranges), cl,
             service::storage_proxy::coordinator_query_options(executor::default_timeout(), std::move(permit), client_state, trace_state)).then(
-            [this, schema, partition_slice = std::move(partition_slice), selection = std::move(selection), attrs_to_get = std::move(attrs_to_get), start_time = std::move(start_time)] (service::storage_proxy::coordinator_query_result qr) mutable {
+            [this, ac, schema, partition_slice = std::move(partition_slice), selection = std::move(selection), attrs_to_get = std::move(attrs_to_get), start_time = std::move(start_time), cl] (service::storage_proxy::coordinator_query_result qr) mutable {
+        auto add_capacity =_stats.capacity.add_consumption(ac, true, cl == db::consistency_level::LOCAL_ONE, false);
```

Comment:
I thought the flag was isQuorum but you set it to cl==LOCAL_ONE instead of cl!=LOCAL_ONE?
Or maybe I got confused. I also can't remember by now what the true and false refer to...